src/you_get/extractors/bilibili.py

```diff
@@ -137,9 +137,18 @@ def prepare(self, **kwargs):
             m = re.search(r'property="og:title" content="([^"]+)"', self.page)
             if m is not None:
                 self.title = m.group(1)
+
         if 'subtitle' in kwargs:
             subtitle = kwargs['subtitle']
             self.title = '{} {}'.format(self.title, subtitle)
+        else:
+            m_pages = re.search(r'"pages":(\[[^\]]+])', self.page)
```

Comment:
This will get a wrong string if page title contains the "`]`" character. A better solution is to match the whole `__INITIAL_STATE__`.